<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEvento</class>
 <widget class="QDialog" name="DialogEvento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/imagens/UFC5.jpg);
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>160</width>
     <height>290</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Nome do Evento:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNomeEvento"/>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Local:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditLocal"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Data:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditData">
      <property name="text">
       <string>Ex: dd/mm/aaaa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Hor√°rio:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditHorario">
      <property name="text">
       <string>Ex: 00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Quantidade de Lutas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditQuantidadeDeLutas"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCriarEvento">
      <property name="text">
       <string>Criar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidgetListaEventos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>95</number>
   </attribute>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Lista de Eventos:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>16</width>
     <height>571</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Lista de Lutadores:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetListaLutadores">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>261</width>
     <height>511</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEditPrimeiroLutador">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSegundoLutador">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Primeiro Lutador:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Segundo Lutador:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMarcarLuta">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Marcar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Evento:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomeEvento">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;-------&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRaunds">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>120</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>120</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Rounds:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>713</x>
     <y>0</y>
     <width>20</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Lista de Lutas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGerirLuta">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGerirLuta">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>430</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Posi√ß√£o da Luta:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>551</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>510</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Aprovada</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reprovada</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cancelada</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Status:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStatusLuta">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLutar">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>470</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Lutar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>121</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Posi√ß√£o do Evento:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditPegaEvento"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonGerirEvento">
      <property name="text">
       <string>Escolher</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="DeletarEvento">
      <property name="text">
       <string>Deletar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Posi√ß√£o:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditEditar"/>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButtonEditar">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButtonSalvar">
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>460</y>
     <width>296</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButtonSalvarLuta">
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButtonEditarLuta">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEditEditarLuta"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButtonDeleteLuta">
      <property name="text">
       <string>Deletar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c80000;&quot;&gt;Posi√ß√£o da Luta:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
